Стилизовал хэдер

Roman3341 opened the pull request:
Стилизовал хедер без иконок и кнопки открытия каталога

Roman3341 committed:
Стилизовал хэдер

Стилизовал хедер без иконок и кнопки открытия каталога

Roman3341 committed:
Стилизовал Хэдер №2

Добавил иконки
выровнял нижний список
выровнял корзину

Без состояний и PP

Komnatnaya commented:
![image](https://github.com/htmlacademy-htmlcss/674811-device-1/assets/20788716/be4679b2-21fc-4785-af51-bd7f203b1579)
сверху должен быть отступ

Komnatnaya commented:
![image](https://github.com/htmlacademy-htmlcss/674811-device-1/assets/20788716/2616a065-7d0d-4167-b9bb-ac07c645c69c)
У лого должна быть сверху "точка" фиолетового цвета.

Komnatnaya commented:
![image](https://github.com/htmlacademy-htmlcss/674811-device-1/assets/20788716/07f37ad3-a1db-4170-b795-af1f748d362d)
Это не псевдоэлемент, цифра внутри должна меняться. Можно реализовать спаном.

Komnatnaya commented:
![image](https://github.com/htmlacademy-htmlcss/674811-device-1/assets/20788716/88a0270e-2f90-46a4-b0ac-6f2832060ed0)
Название шрифта оборачиваем кавычками, после веб-безопасного шрифта нужно добавить тип семейства

Komnatnaya commented:
Форма не должно отправляться пустой, нужно сделать поле ввода обязательным.

Komnatnaya commented on styles/styles.css:
```diff
@@ -0,0 +1,425 @@
+}
+
+.visually-hidden {
+  position: absolute;
```
```
.visually-hidden {
  position: absolute;
  width: 1px;
  height: 1px;
  margin: -1px;
  border: 0;
  padding: 0;
  white-space: nowrap;
  clip-path: inset(100%);
  clip: rect(0 0 0 0);
  overflow: hidden;
}
```

Komnatnaya commented:
![image](https://github.com/htmlacademy-htmlcss/674811-device-1/assets/20788716/ce9b3e1d-1b61-466a-ba66-e259f50f36fe)
Стилизовать шрифт в инпуте и кнопке, по умолчанию они не наследуют шрифт от родителя.

Roman3341 committed:
Исправление замечаний

Замечания по коммиту "Стилизовал Хэдер№2"

Roman3341 committed:
Стилизовал index

Не сделал только модальное окно специального заказа и кнопки слайдера.
Состояния еще не трогал

Komnatnaya commented:
![image](https://github.com/user-attachments/assets/8619de37-8994-4cc7-88e7-c9a4eb916993)
не тот шрифт в инпуте и в кнопке

Komnatnaya commented:
![image](https://github.com/user-attachments/assets/a6b231b6-81cc-4e29-a97e-e32e783ced16)
переставила активный класс на другой слайд

Komnatnaya commented:
![image](https://github.com/user-attachments/assets/263fa51b-b772-44f1-b786-c47c0f9699e3)
на третьем

Komnatnaya commented:
![image](https://github.com/user-attachments/assets/9fb7d5d8-cd4c-4e5b-98d5-7e53b2bcae1b)
телефон должен быть на отдельной строке, возможно его стоит вынести из абзаца.

Roman3341 committed:
Сделал модальное окно

Окно заказа редкого товара

Komnatnaya commented on index.html:
```diff
@@ -361,6 +371,37 @@ <h2 class="visually-hidden">Подписка на рассылку</h2>
+              <label class="rare-modal__label rare-modal__label--count" for="count">Количество:</label>
+              <button type="button" class="rare-modal__button-count rare-modal__button-count--minus"></button>
+              <input type="number" id="count" class="rare-modal__input rare-modal__input--count">
+              <button type="button" class="rare-modal__button-count rare-modal__button-count--plus"></button>
```
Стоит добавить подписи кнопкам -/+

Komnatnaya commented:
![image](https://github.com/user-attachments/assets/e7ac3d72-6f4d-46d5-9e84-85b73cca6d64)
У кнопки все еще не тот шрифт, там должен быть Рубик

Komnatnaya commented:
![image](https://github.com/user-attachments/assets/bac12631-fdb0-4bb8-a6fd-c6f5f9e34428)
нужно удалить стрелочки, их заменяют кнопки -/+

Komnatnaya commented:
![image](https://github.com/user-attachments/assets/168f9c64-5c25-457d-a8cf-2467b86f0ce3)
инпут можно сразу поставить в состоянии ошибки и показать подпись, добавив класс ошибки инпуту или его обертке

Komnatnaya commented:
![image](https://github.com/user-attachments/assets/cb8ae496-8709-446c-8b0d-b295e0345764)
не хватает иконок

Komnatnaya commented:
![image](https://github.com/user-attachments/assets/79ffc355-2152-49dc-a2cc-ff157701ab5b)
стоит добавить value

Komnatnaya commented:
![image](https://github.com/user-attachments/assets/19d18aae-98d0-4e06-b73f-58646f92dddd)
оверлей по макету светлый с прозрачностью, а не темный

Komnatnaya commented:
![image](https://github.com/user-attachments/assets/eb1ad131-f7f8-4549-bdc0-bb81cc0cfa1a)
Не хватает тултипа

Komnatnaya commented:
![image](https://github.com/user-attachments/assets/6d888e64-0363-440c-a75d-c0668a19ad71)
z-index

Komnatnaya commented:
![image](https://github.com/user-attachments/assets/9969b600-dac7-47c3-97da-4f890afa254b)
и тут

Roman3341 committed:
Исправил замечения

Komnatnaya commented:
![image](https://github.com/user-attachments/assets/6a1d8027-3d00-449d-ab40-5c210240eea6)
по макету бордер 2рх
